ccd-definition/CaseEvent.json

```diff
@@ -283,6 +283,20 @@
     "ShowEventNotes": "N",
     "EndButtonLabel": "Delete application"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "otherProposalDecision",
+    "Name": "Allocation decision",
+    "Description": "Entering other proceedings and allocation proposals",
+    "DisplayOrder": 19,
```

Comment:
is 19 the correct display order for this? 

I guess eventually this will fit into a different state entirely, so maybe not important

Reply:
as this is the first event to be displayed for gatekeeper, you are right, i've changed it to 1 but it will not affect anything as of yet :)